### ISSUE TITLE
Page content missing when JS is disabled

### ISSUE THREAD

**Issue by bjorn3** [rust-adventure/thisweekinbevy#6]

**Comment by ChristopherBiscardi** [comment]
I don't plan to support JavaScript being disabled for the sake of JavaScript being disabled. The vast majority of the internet breaks when JavaScript is disabled.

**Comment by bjorn3** [comment]
> The vast majority of the internet breaks when JavaScript is disabled.

Not at all. The vast majority of static pages (like this is) works just fine without javascript. Sure, web apps do break, but that is to be expected. For static pages there is no excuse however. Whatever the javascript code does could have been done before sending any HTML to the browser.

> JavaScript being disabled for the sake of JavaScript being disabled.

I block it by default to significantly reduce exposure to trackers or websites that think it is funny to show your mouse position to all other visitors of the page without asking for consent (yes, that happened to me a couple of days ago. I only found out after it showed me the cursor of someone else. I made the mistake to allow js for this site for something that genuinely needed js)
